app/controllers/authenticate_controller.rb

```diff
@@ -39,9 +39,9 @@ def index
   end
 
   def status
-    Authentication::ValidateStatus.new.(
-      authenticator_status_input: status_input,
-      enabled_authenticators: Authentication::InstalledAuthenticators.enabled_authenticators_str
+    Authentication::Handler::StatusHandler.new.call(
+      status_input: status_input,
+      enabled_authenticators: Rails.application.config.conjur_config.authenticators
```

Comment:
I know it's not actively promoted, but this specifically deprecates the authenticators in the database config. Which makes this authenticator behave differently from the others.

Would it not be better to switch this to `Authentication::InstalledAuthenticators.enabled_authenticators` instead to get the array, rather than a string?